packages/block-library/src/index.native.js

```diff
@@ -124,6 +129,7 @@ const registerBlock = ( block ) => {
 // eslint-disable-next-line no-undef
 const devOnly = ( block ) => ( !! __DEV__ ? block : null );
 
+const iOSOnly = ( block ) => ( Platform.OS === 'ios' ? block : null );
```

Comment:
We still want the pullquote block showing up on dev builds, but this is entirely disabling it on Android, right?

Reply:
Good call, let me update the code